src/chrome/content/rules/prnt.sc.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="prnt.sc" platform="mixedcontent">
+	<target host="prnt.sc" />
+	<target host="www.prnt.sc" />
+	<target host="ads.prnt.sc" />
+		<test url="http://ads.prnt.sc/ads/adtrue/728x90/index.html" />
```

Comment:
done. this `ruleset` is longer a `default_off`, it is safe to merge this regardless of the result of #9906. thanks.